tests/api/v2/handlers/test_abilities_api.py

```diff
@@ -0,0 +1,142 @@
+import pytest
+
+from http import HTTPStatus
+
+from app.objects.c_ability import Ability
+from app.objects.secondclass.c_executor import Executor, ExecutorSchema
+from app.utility.base_service import BaseService
+
+
+@pytest.fixture
+def test_ability(loop, api_v2_client, executor):
+    executor_linux = executor(name='sh', platform='linux')
+    ability = Ability(ability_id='123', name='Test Ability', executors=[executor_linux],
+                      technique_name='collection', technique_id='1')
+    loop.run_until_complete(BaseService.get_service('data_svc').store(ability))
+    return ability
+
+
+class TestAbilitiesApi:
+    async def test_get_abilities(self, api_v2_client, api_cookies, test_ability):
+        resp = await api_v2_client.get('/api/v2/abilities', cookies=api_cookies)
+        abilities_list = await resp.json()
+        assert len(abilities_list) == 1
+        ability_dict = abilities_list[0]
+        assert ability_dict['ability_id'] == test_ability.ability_id
```

Comment:
would it be more efficient to do `assert ability_dict == test_ability.schema.dump(test_ability)` or something like that?

Reply:
If that works, you could use that sort of comparison in the other tests, as well

Reply:
Refactored a bit - now there are payload fixtures for the replace, create, and update tests as well. These are now also used for dictionary comparisons